### ISSUE TITLE
sublist3r api is no longer available

### ISSUE THREAD

**Issue by SharpSeeErEPlace** [jonluca/Anubis#93]
# Please follow the general troubleshooting steps first:

- [x] Check you are using Python 3.6
- [x] Make sure you are using pip3 (or any instance of pip associated with Python 3.x)

### Bug reports:

Make sure you include the following:

* Anubis Version  1.1.3
* Python Version  3.11.8
* Full stack trace if crash or installation error  N/A
* Machine details (OS, build number, distro, etc.)  Pop!_OS 22.04 LTS

The default scan for anubis includes searching `sublist3r` but gives an error:  `Exception when searching sublist3r`.  Looking at the [code](https://github.com/jonluca/Anubis/blob/master/anubis/scanners/sublist3r.py#L13): 
```python
results = requests.get('https://api.sublist3r.com/search.php', headers=headers, params=params)
```

It's calling to `api.sublist3r.com`, but the `sublist3r.com` domain name is now owned by some random domain parking company (pinging it resolves to `lb01.parklogic.com`).

I would suggest removing sublist3r as a resolution tool in Anubis.

P.S. Fantastic tool!  Thanks so much!

**Comment by jonluca** [comment]
Thanks for the report! Will remove sublister as a source in the next update

**Comment by jonluca** [comment]
Fixed in v1.2.0
